app/src/main/play/listings/en-US/full-description.txt

```diff
@@ -3,7 +3,7 @@ allowing you to make changes to it as well.
 Especially useful when you need to open an external link either from an email, a social network app or others.
 
 
-Url Checker is developed by TrianguloY originally for personal use, it is open source (under CC BY 4.0 license), free, without any ads or trackers, light sized and using as few permissions as workable (only the internet permission, for module checks that will only be performed when the user request them). The source code is available on GitHub too if you want to suggest changes, perform modifications or propose translation: https://github.com/TrianguloY/UrlChecker
+Url Checker is developed by TrianguloY, originally for personal use. It is open source (under CC BY 4.0 license), free, without any ads or trackers, light sized and using as few permissions as necessary (only the internet permission, for module checks that will only be performed when the user request them). The source code is available on GitHub too, if you want to suggest changes, perform modifications or propose translations: https://github.com/TrianguloY/UrlChecker
```

Comment:
It's not really useful to have the word "translations" in the plural, in my opinion (one person -> one translation, most of the time)

Reply:
Hmm, most of the time, but on my Click to Chat app I think there wer 1 or 2 users that gave me two translations each.
For singuular you need to add 'a' (I think).
Hmm, what about:

`... perform modifications or provide a new translation: https://github.com/TrianguloY/UrlChecker`

Reply:
very good, it's not a critical point